nucliadb/nucliadb/search/tests/unit/search/search/chat/test_query.py

```diff
@@ -42,28 +36,6 @@ def predict():
         yield predict
 
 
-async def test_chat_does_not_call_predict_if_no_find_results(
```

Comment:
This feature is no longer wanted?
AFAIR if we send a question to the LLM without context it is more likely to do hallucinations.